misc/docerina/build.gradle

```diff
@@ -16,7 +16,7 @@
  */
  
 apply plugin: 'base'
-apply plugin: 'com.github.johnrengelman.shadow'
+//apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
Why we do not need this plugin?

Reply:
We removed the shadowJar task from there